### PR TITLE
fix apex and e2e tests on CI

### PR DESCRIPTION
**Pull request by rohit21agrawal**
configure our ci pipeline for apex and e2e so it doesn't need corp access anymore.

### CODE REVIEW

File: build/vsts_build.yaml [hunk 1]
```diff
@@ -279,6 +269,14 @@ phases:
         $(VsixPublishDir)\\EndToEnd.zip 
       TargetFolder: "$(BuildOutputTargetPath)\\artifacts"
 
+  - task: PublishBuildArtifacts@1
+    displayName: "Publish NuGet.exe VSIX and EndToEnd.zip as artifact"
+    inputs:
+      PathtoPublish: "$(Build.Repository.LocalPath)\\artifacts\\$(VsixPublishDir)"
```
Comment (nkolev92) [inline build/vsts_build.yaml:275]: Where does this point to?

Comment (rohit21agrawal) [reply build/vsts_build.yaml:275]: $(RepoRoot)\artifacts\VS15

**Review by nkolev92 — APPROVED** [review]

File: scripts/e2etests/NuGetFunctionalTestUtils.ps1 [hunk 2]
```diff
@@ -215,36 +210,30 @@ function CopyResultsToCI
...
 
-function CopyActivityLogToCI
-{    
-    if($env:ActivityLogFullPath) 
-    {
-        Write-Host "Copying activity log file from $env:ActivityLogFullPath to $env:EndToEndResultsDropPath"
-        Copy-Item $env:ActivityLogFullPath -Destination $env:EndToEndResultsDropPath -Force  -ErrorAction SilentlyContinue
-    }
-}
+# function CopyActivityLogToCI
```
Comment (mishra14) [inline scripts/e2etests/NuGetFunctionalTestUtils.ps1:229]: why did we comment this out?

**Review by mishra14 — APPROVED** [review]

File: scripts/e2etests/NuGetFunctionalTestUtils.ps1 (hunk 2 above)
Comment (rohit21agrawal) [reply scripts/e2etests/NuGetFunctionalTestUtils.ps1:229]: hasn't really been helpful, and don't want to spend too much time trying to make this work. will un-comment if we find a need for it later.

**Pull request merged**